design-docs/CLI-Subscriptions.md

```diff
@@ -237,11 +237,11 @@ Options:
 
 ```
 planet subscriptions request \
```

Comment:
May I suggest having an example that includes all of the options? In this case, from the docstring alone it's unclear to me what an appropriate input would be for `rrule`.

Reply:
Great suggestion. However, since this is just a design doc, and not actual user-faced documentation, we don't need to get into documenting every option so I won't go into rrule for now.